src/backend/_scss/application.scss

```diff
@@ -522,39 +522,63 @@ a {
   min-height: 255px;
   position: absolute;
   transition: all 0.5s ease;
+  background-color:white;
   width: 355px;
   z-index: 1000;
 
-  .pop {
-    background-color: $white;
+    .pop>hr {
+      margin: 10px 0px 10px 0px;
+      width: calc(100% - 20px)!important;
+    }
 
-    .pop-over {
-      color: $gray-dark;
-      font-weight: 400;
-      line-height: 20px;
-      margin: 10px;
-      padding: 2%;
-      word-wrap: break-word;
+    .pop>h4 {
+      margin-left: -20px;
+    }
+
+    .pop>.pop-over.row {
+      padding: 10px 20px 0px 0px;
+    }
+
+    .pop {
+      background-color: #fff;
+      margin-left: 20px;
+
+        a {
+          margin-left: -5px;
+        }
+
+        .pop-over>.col-xs-12, .pop-over>.col-md-12 {
```

Comment:
Don't refer col-md-12 like this. Instead make a class and refer like .pop-over>classname